packages/synapse-interface/constants/tokens/bridgeable.ts

```diff
@@ -1180,3 +1181,19 @@ export const USDB = new Token({
   priorityRank: 103,
   routeSymbol: 'USDB',
 })
+
+export const SPECTRAL = new Token({
+  priorityRank: 600,
+  addresses: {
+    [CHAINS.BASE.id]: '0x96419929d7949D6A801A6909c145C8EEf6A40431',
```

Comment:
will need Ethereum address/decimals in here and `synapse-constants` when available.